owners/test/owners_bot.test.js

```diff
@@ -26,20 +28,71 @@ const {
 
 describe('owners bot', () => {
   let sandbox;
-  const fakeGithub = new GitHub({}, 'ampproject', 'amphtml', console);
-  const pr = new PullRequest(1337, 'author_user', 'test_sha');
+  const github = new GitHub({}, 'ampproject', 'amphtml', console);
```

Comment:
The handful of changes in these test files are here because tests from two different files used similar constants (like the PR), so now that tests share one instance they are getting updated to reflect that. No functional change.